src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -27,6 +28,7 @@
     public static final String DEFAULT_FORMCLASS = "6E1";
     public static final String DEFAULT_GRADES = "123.0";
     public static final String DEFAULT_POSTALCODE = "123456";
+    public static final String DEFAULT_REMARK = "";
```

Comment:
Perhaps you can have some random remark to have consistency with the other DEFAULT strings